107/src/test/java/org/ehcache/jsr107/ConfigurationMergerTest.java

```diff
@@ -273,6 +278,53 @@ public void jsr107LoaderInitAlways() {
     assertThat(configHolder.useEhcacheLoaderWriter, is(false));
   }
 
+  @Test
+  public void jsr107DefaultEh107IdentityCopierForImmutableTypes() {
+    XmlConfiguration xmlConfiguration = new XmlConfiguration(getClass().getResource("/ehcache-107-default-copiers.xml"));
+    final DefaultJsr107Service jsr107Service = new DefaultJsr107Service(ServiceLocator.findSingletonAmongst(Jsr107Configuration.class, xmlConfiguration.getServiceCreationConfigurations().toArray()));
+    merger = new ConfigurationMerger(xmlConfiguration, jsr107Service, mock(Eh107CacheLoaderWriterProvider.class));
```

Comment:
This test is about XML config without any default copiers provided for the immutable types, so what else is missing.
No XML config:- all the JSR caches are usable through XML config only, or you want mock based tests when you say No XML config.